src/DynamicRupture/FrictionLaws/ImposedSlipRates.h

```diff
@@ -12,17 +12,15 @@ class ImposedSlipRates : public BaseFrictionLaw<ImposedSlipRates<STF>> {
   public:
   using BaseFrictionLaw<ImposedSlipRates>::BaseFrictionLaw;
 
-  real (*slip1)[misc::numPaddedPoints];
-  real (*slip2)[misc::numPaddedPoints];
+  real (*imposedSlipDirections)[misc::numPaddedPoints][2];
```

Comment:
I like previous solution. Why couldn't we make `imposedSlipDirections1` and `imposedSlipDirections2`

Here are some comments in case if a simple renaming is not going to work. `real (*imposedSlipDirections)[misc::numPaddedPoints][2]` is going to reside in a non-unit stride access.
Maybe,  we could make `real (*imposedSlipDirections)[2][misc::numPaddedPoints];`

I would also propose to add some enum to access data in each direction 
```
enum Direction {
  One = 0,
  Two,
  Count
}
```